samtranslator/model/api/http_api_generator.py

```diff
@@ -511,6 +513,61 @@ def _add_tags(self) -> None:
         open_api_editor.add_tags(self.tags)
         self.definition_body = open_api_editor.openapi
 
+    def _get_permission(
+        self, authorizer_name: str, authorizer_lambda_function_arn: str, api_arn: str
+    ) -> LambdaPermission:
+        """Constructs and returns the Lambda Permission resource allowing the Authorizer to invoke the function.
+
+        :returns: the permission resource
+        :rtype: model.lambda_.LambdaPermission
+        """
+
+        partition = ArnGenerator.get_partition_name()
+        resource = "${__ApiId__}/authorizers/*"
+        source_arn = fnSub(
+            ArnGenerator.generate_arn(partition=partition, service="execute-api", resource=resource),
```

Comment:
We shouldn’t hardcode the partition or linters will complain. Better to use the pseudo-parameter.